### DISCUSSION TITLE
Support for OpenAPI Paths

### DISCUSSION THREAD

**Discussion by WoobaWooba** [cue-lang/cue#1741] [Q&A] [answered]
From what I've researched, CUE still has no support for OpenAPI paths, as mentioned here https://github.com/cue-lang/cue/discussions/1293

Are there any plans in the future to implement this feature? If so, when can we roughly expect it?

**Comment by myitcv** [comment, marked as answer]
https://github.com/cue-lang/cue/issues/735 is probably the best place to have this discussion. Thanks
